scripts/next.coffee

```diff
@@ -0,0 +1,22 @@
+# Description:
+#   Responds to `!next` with a helpful message.
+#
+# Dependencies:
+#   none
+#
+# Configuration:
+#   HUBOT_NEXT_MESSAGE: The message to repond to the trigger with. Defaults to "Another Satisified Customer! NEXT!"
+#
+# Commands:
+#   !next - responds with the message
+#
+# Author:
+#   Ari Porad (@ariporad) <ari@ariporad.com>
+#
+# License:
+#   MIT License: https://ariporad.mit-license.org
+#
+
+module.exports = (robot) ->
+  robot.hear /\!next/i, (res) ->
```

Comment:
I don't really think you're going to succeed with that. Assuming all of these are valid:

> And you should blow up the moon. Problem Solved. !next

> And you should blow up the moon. Problem Solved. Who's !next?

> !next

Then that won't work. It might just have to be dumb.